<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DecypherDialog</class>
 <widget class="QDialog" name="DecypherDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>282</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>374</width>
    <height>282</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DecypherDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
    <item>
      <widget class="QLabel" name="choosenFileLabel">
       <property name="text">
        <string notr="true">No file choosen</string>
       </property>
      </widget>
     </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="cypheredFileButton">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Choose encrypted file</string>
     </property>
    </widget>
   </item>
   <item>
      <widget class="QLabel" name="keyPathLabel">
       <property name="text">
        <string notr="true">No key choosen</string>
       </property>
      </widget>
     </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="importKeyButton">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>Import file key</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="decypherButton">
     <property name="minimumSize">
      <size>
       <width>290</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>290</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Decypher</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
 <connection>
   <sender>cypheredFileButton</sender>
   <signal>clicked()</signal>
   <receiver>DecypherDialog</receiver>
   <slot>onImportFileButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>importKeyButton</sender>
   <signal>clicked()</signal>
   <receiver>DecypherDialog</receiver>
   <slot>onImportKeyButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>decypherButton</sender>
   <signal>clicked()</signal>
   <receiver>DecypherDialog</receiver>
   <slot>onDecypherButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
